### DISCUSSION TITLE
Can commit dialog show pending changes as a tree?

### DISCUSSION THREAD

**Discussion by SetoKaiba** [gitextensions/gitextensions#8706] [Q&A]
Can commit dialog show pending changes as a tree?
It's hard to select the files in one folder without this feature.

**Comment by pmiossec** [comment]
No, that's not possible.
As a workaround, you could filter on the folder name.

**Comment by ghost** [comment]
I think this is a good idea to implement. All lists must support the viewport as a list and as a tree.
